### ISSUE TITLE
Use libdns instead of lego

### ISSUE THREAD

**Issue by mattiasb** [mdbraber/acme-proxy#12]
Hi!

Have you looked at using [libdns](https://github.com/libdns/libdns) over lego for solving the dns challenge? It's the same library as `certmagic` uses so you're already depending on it. :D

**Comment by mdbraber** [comment]
@mattiasb yes I have and have contributed even a TransIP module to libdns. Unfortunately I don't have time to currently work on acmeproxy, but happy to accept a PR

**Comment by mattiasb** [comment]
> @mattiasb yes I have and have contributed even a TransIP module to libdns. Unfortunately I don't have time to currently work on acmeproxy, but happy to accept a PR

Cool, we're going to need something very similar at my workplace so I'm kind of weighing my options. :)
